app/forms/mentee_application_state_form.rb

```diff
@@ -0,0 +1,21 @@
+class MenteeApplicationStateForm
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+  include ActiveModel::Validations
```

Comment:
I'm probably including more than I need to. Can remove to barebones, since form isn't being used in the create controller action.